submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/SubmissionDatesTest.kt

```diff
@@ -0,0 +1,364 @@
+                        line("Title", "Sample Submission")
+                        line("ReleaseDate", "2011-04-24")
+                        line()
+
+                        line("Study")
+                        line()
+                    }.toString()
+
+                assertThatExceptionOfType(WebClientException::class.java)
+                    .isThrownBy { userWebClient.submitSingle(v2, TSV) }
+                    .withMessage(
+                        """
+                        {
+                          "log": {
+                            "level": "ERROR",
+                            "message": "Submission validation errors",
+                            "subnodes": [{
+                              "level": "ERROR",
+                              "message": "The release date of a public study cannot be changed",
+                              "subnodes": []
+                            }]
+                          },
+                          "status": "FAIL"
+                        }
+                        """.trimIndent(),
+                    )
+            }
+    }
+
+    @Nested
+    inner class AdminUser {
+        private lateinit var adminWebClient: BioWebClient
+
+        @BeforeAll
+        fun init(): Unit =
+            runBlocking {
+                securityTestService.ensureUserRegistration(SuperUser)
+                adminWebClient = getWebClient(serverPort, SuperUser)
+            }
+
+        @Test
+        fun `28-6 Admin Submit and Re Submit in the past`() =
+            runTest {
+                val v1 =
+                    tsv {
+                        line("Submission", "RELEASE-0010")
+                        line("Title", "Sample Submission")
+                        line("ReleaseDate", "2020-04-24")
+                        line()
+
+                        line("Study")
+                        line()
+                    }.toString()
+
+                assertThat(adminWebClient.submitSingle(v1, TSV)).isSuccessful()
+                val v1Extended = submissionRepository.getExtByAccNoAndVersion("RELEASE-0010", 1)
+                assertThat(v1Extended.releaseTime).isEqualTo(OffsetDateTime.parse("2020-04-24T00:00+00:00"))
+
+                val v2 =
+                    tsv {
+                        line("Submission", "RELEASE-0010")
+                        line("Title", "Sample Submission")
+                        line("ReleaseDate", "1985-04-24")
+                        line()
+
+                        line("Study")
+                        line()
+                    }.toString()
+                assertThat(adminWebClient.submitSingle(v2, TSV)).isSuccessful()
+                val v2Extended = submissionRepository.getExtByAccNoAndVersion("RELEASE-0010", 2)
+                assertThat(v2Extended.releaseTime).isEqualTo(OffsetDateTime.parse("1985-04-24T00:00+00:00"))
+            }
+
+        @Test
+        fun `28-7 Admin make a public submission private`() =
```

Comment:
We have a couple of tests to cover this in ResubmissionApiTest. We should remove or migrate those here since they're date related. Specifically 

5-4     | regular user suppresses own submission
5-5     | super user suppresses submission from another user